### PR TITLE
Producer improvement

### PR DESCRIPTION
**Pull request by kobergj**

### CODE REVIEW

File: producing/models/universe.py [hunk 1]
```diff
@@ -127,3 +133,24 @@ def update(self):
...
+            distance += (x - y)**2
+
+        distance = math.sqrt(distance)
+        distance = round(distance, 2)
+
+        return distance
+
+    def calculateTravelCosts(self, Player, Distance):
+        # Check if reachable
+        if Distance <= Player.currentShip.maxTravelDistance():
```
Comment (ivan-kostko) [inline producing/models/universe.py:155]: What would be returned, if not a true?

File: consuming/visualization/terminal_viz.py [hunk 2]
```diff
@@ -0,0 +1,367 @@
+import logbook.configuration as log
+
+class MainScreen():
+    # Where to store them best? 7   '0123456'
+    mapIdentifiers = {'Empty':      '       ',
```
Comment (ivan-kostko) [inline consuming/visualization/terminal_viz.py:5]: Wouldn't it be better to move such a mapping into config and have an instance containing this strings bounded as properties? Then, you would have type check/safety at compilation time avoiding missing map entries at execution time.

Comment (ivan-kostko) [inline consuming/visualization/terminal_viz.py:5]: So, MainScreen should have just a few properties and methoods like Render(), CleanScr(), and so on... Also, if you want to follow MVCVM pattern, or stick close to it, it MUST have its own Model and ViewModel. Then, such a mapping would move to MainScreenModel, or even better, instance of the class mentioned in previous comment should be referencied from MainScreenViewModel.
Nevertheless, Im not sure that MainScreen class has anything in common with Details representation. From my point of view, MainScreen should operate on Windows and take care only about them. Thus, the Window MVCVM would take care about rendering Detail depending on for.ex. Zoom, window status and so on. In this case, the base class WindowViewModel should have a reference to INDEPENDENT instance of  some class which provides/serves such a mapping as properties.

**Comment by ivan-kostko** [comment]
Try to google MVCVM design pattern. Or assemble a few good explanations from this BS thread https://www.3dbuzz.com/forum/threads/193704-I-need-help-with-MVVM-pattern-Or-more-specifically-the-MVCVM-pattern

**Comment by ivan-kostko** [comment]
It is slightly independent on PR topic, but missleading of general defenitions of the pattern is very visible through the code.

**Comment by kobergj** [comment]
Moved to New Branch 'InsertViewModels'

**Pull request merged**